src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -61,11 +61,19 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   const fundingMode = formData.proposalType === FUNDING_PROPOSAL
 
-  const forumRegex = new RegExp(connectedGarden.forumURL)
+  // Escaping forumURL to avoid misuse with regexp
+  const forumURLRegex = connectedGarden.forumURL.replace(
```

Comment:
I'm assuming forumURL it's required, will always be there, its correct?

Reply:
What if the connectedGarden.forumURL is null? But if it is required then there is no need for more validation 👌